workflows/utilities/wf_organism_parameters.wdl

```diff
@@ -48,7 +49,36 @@ workflow organism_parameters {
     Float? narrow_bandwidth
     Float? proportion_wide
   }
-  if (organism == "sars-cov-2" || organism == "SARS-CoV-2") {
+  # for morgana_magic & theiameta_panel compatibility
+  if (defined(taxon_id)) {
```

Comment:
To me, this might almost be better housed in a vsp-specific task in a "panel workflow." Takes in the taxon_id & panel name, and then spits out the corresponding string value, which is then passed to wf_organsim_parameters. Then, we wouldn't touch organism parameters at all, and we could have a separate "panel workflow" that would define these values if we end up expanding this to function with other panels. Let's discuss further.

Reply:
Good idea; i am a believer